docs/manuals/getting-started/index.rst

```diff
@@ -0,0 +1,24 @@
+Getting Started
+===============
```

Comment:
The new manual/ directory is fine for now to help with the wholesale move, but I ask you to please reconsider later the division of docs/ into subdirectories. I don't know how anybody can know if a document they are looking for is in manuals/ or guides/ or what's the difference between the two, and the difference between those two to design-notes.

Reply:
@nyh - in going forward we have two choices
1 - keep the two doc types apart from each other.
2 - integrate them

As I discovered with this pilot PR, I am more in favor of #2. Let me know what you think

Reply:
My point is that maybe (?) it makes sense to "keep the two doc types apart", but the problem is that we don't just have two doc types - we already had (after a former move that I never liked) subdirectories like design-notes/ and guides/ (and also contribute/ and alternator/) and now we're adding a fifth subdirectory manuals/. The problem is not the fact that we have 5 subdirectories - it's that it's not clear to users or to developers what belongs where... What's the difference between "manuals" and "guides", for example? As I said, I don't think we need to do anything about this right now, but it's something we'll need to consider in the future. I think we should not continue to have two subdirectories "manuals" and "guides" - either one will need to be renamed, or these two vague subdirectories can be "opened up" to many subdirectories replacing those two.